### ISSUE TITLE
Include an option to download gaf and phaf from gene pages

### ISSUE THREAD

**Issue by ValWood** [pombase/website#1655]
https://github.com/pombase/website/issues/949

We would like to be able to download tabulated annotation from gene pages.. Because we can.

**Comment by kimrutherford** [comment]
How should this work?  Do we want a separate link under each section (MF, BP, CC and phenotype) that just downloads the annotations of that type?

That's quite easy to do for GAF export because the existing code can be mostly re-used.  The phenotype export needs new code.  Also it will only work for single allele genotype annotations which might seem inconsistent for the users.

**Comment by ValWood** [comment]
I guess a separate link for each will be fine. I don't have any strong feelings for this..

**Comment by kimrutherford** [comment]
I'm not sure either.  I guess that other option is to have a separate "Annotation download" section where you can select what you need.

I'll leave this for a few weeks so we can think about it.

**Comment by ValWood** [comment]
Putting as future until we have a specific use case. Much more important stuff first...

**Comment by ValWood** [comment]
I don't see the point in implementing this. I think it was suggested on  one of our group meetings, but if people want to do this  they can do it via the download (at least for GO). We can direct people to [pombemine if they ever ask for phenotypes. Closing unless we are specifically asked for this.

**Comment by kimrutherford** [comment]
> I don't see the point in implementing this. I think it was suggested on one of our group meetings, but if people want to do this they can do it via the download (at least for GO). We can direct people to [pombemine if they ever ask for phenotypes. Closing unless we are specifically asked for this.

What changed your mind?

**Comment by ValWood** [comment]
I just thought it wasn't high priority, and could be done via the 
"single gene list in advanced search"
I also can't think of any particular  use case.

that doesn't mean we can't do it if it's easy. Its extra links on the page though for not much benefit.


maybe it is useful for genes with lots of phenotypes.

**Comment by ValWood** [comment]
actually, I can't see PHAF as a download option in "Advanced...."

**Comment by kimrutherford** [comment]
> actually, I can't see PHAF as a download option in "Advanced...."

I never got around to implementing that.  Probably makes sense to add it?  If so, I'll make an issue.  We'll should make it clear in the download dialog that PHAF files only contain single allele phenotypes.

Once we have a [multi-allele PHAF format](/pombase/pombase-chado/issues/496) we could think about downloading the multi-allele phenotypes too.

**Comment by ValWood** [comment]
Yes, nobody has ever asked for it, but I guess we can make it available since the code exists.
